pennylane_lightning/core/src/simulators/lightning_kokkos/gates/GateFunctorsGenerator.hpp

```diff
@@ -847,7 +847,7 @@ template <class PrecisionT, bool adj = false> struct generatorCRXFunctor {
 
         arr[i00] = Kokkos::complex<PrecisionT>{0.0, 0.0};
         arr[i01] = Kokkos::complex<PrecisionT>{0.0, 0.0};
-        swap(arr[i10], arr[i11]);
+        kokkos_swap(arr[i10], arr[i11]);
```

Comment:
Is there any other "new" or "updated" feature that we are interested in from Kokkos v4.3.1?

Reply:
Nothing to worry about just yet --- we get updated support, bugfixes, and the removal of the deprecated KE swap option.